src/main.rs

```diff
@@ -83,6 +83,7 @@ async fn main() -> Result<()> {
             (@subcommand inspect =>
              (about: "Inspect Kubewarden policy")
              (@arg ("uri"): * "Policy URI. Supported schemes: registry://, https://, file://")
+             (@arg ("output"): -o --("output") +takes_value "output format. One of: yaml")
```

Comment:
Hm, seems that the `clap_app!` macro does not have a way to provide a set of valid values. I can check later if there's a nice way we can do this.